pkg/auth/authenticator/password/htpasswd/htpasswd.go

```diff
@@ -116,82 +115,3 @@ func (a *Authenticator) loadIfNeeded() error {
 	}
 	return nil
 }
-
-func testPassword(password, hash string) (bool, error) {
```

Comment:
Ok, I see you need those tests in the authentication. Will update this with just the body closing